.github/workflows/ci-cd.yml

```
@@ -334,6 +334,13 @@ jobs:
         --skip-missing-interpreters false
         --notest
 
+    - name: Drop Git tags from HEAD for non-tag-create events
+      run: >-
+        git tag --points-at HEAD
+        |
+        xargs git tag --delete
```

What's the point in wrapping this one-liner to multiline?
For me it looks odd when split to three lines 🤷🏻 
```suggestion
      run: git tag --points-at HEAD | xargs git tag --delete
```

---

Because it shows separate chained commands more clearly. At it's also synched from other projects. The desire to diverge from that boilerplate was what contributed to this being missed in the original integration.

I sometimes do 
```yaml
run: >-
  cmd1
  | cmd2
  | cmd3
```
and it works well.

But in this case, I've gone more for more lines.

I think that the synchronized changes should be kept as is, as much as possible, to avoid similar problems. It also allows running a `diff` against files in two repositories when synchronizing new changes. I often do an `nvim -d` to copy bits over. The more divergence there is, the harder it is to figure out what to sync and what should be disregarded.

So I suggest not reformatting this.

---

> Because it shows separate chained commands more clearly.

Not for me 🤷🏻 This bit of code made me stop by and think wth is this about.

> I sometimes do

This one is a way better I think. Even despite I see no value in wrapping small one-liners into multiline that distracts attention from what the command does. But that's my personal opinion and preference vs you two 😂

---

I preffer oneliner, but webknjaz have plans to improve reusable workflow during 2025, so let it as is for now, for simplifying his future work. If improvements do not come, we will refactor it in a year to what we want :)

---

Works for me 🤝